src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -120,6 +120,8 @@
     'geoid_height_above_reference_ellipsoid': ('geoidUndulation', 'float'),
     'gnss_sat_class': ('satelliteConstellationRO', 'string'),
     'impact_height': ('impactHeightRO', 'float'),
+    'height': ('impactHeightRO', 'float'),
+    'Height': ('impactHeightRO', 'float'),
```

Comment:
I thought these lines would take the key in the file `Height` and then map it to `MetaData/impactHeightRO` but this did not work so was done brute force below

Reply:
hmm, yeah it should've done that, weird...